gallery/src/components/NxList/NxListPage.tsx

```diff
@@ -0,0 +1,284 @@
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Link</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This list element behaves as a link, i.e. an <NxCode>{'<a>'}</NxCode> tag, and requires
+                a <NxCode>{'href'}</NxCode> prop. This element is also clickable, responds to hover events,
+                and has selected and disabled states.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Text</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList.Item</NxCode>, <NxCode>NxList.Button</NxCode>,
+                or <NxCode>NxList.Link</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>
+                The primary text content of the list item, displayed in a heavier font weight.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Subtext</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList.Item</NxCode>, <NxCode>NxList.Button</NxCode>,
+                or <NxCode>NxList.Link</NxCode>
+              </NxTable.Cell>
+              <NxTable.Cell>
+                The secondary text content of the list item displayed as non-bolded text.
+                The subtext is displayed below the bolded, main list item text.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Actions</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList.Item</NxCode></NxTable.Cell>
+              <NxTable.Cell>A container for buttons inside list items. Use this when you want to have
+                a button/buttons on the far right.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Empty</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>Used when there are no list items returned. In this case it's expected that there
+                would only be a single list item which contains the empty element.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Error</NxCode></NxTable.Cell>
+              <NxTable.Cell>Within <NxCode>NxList</NxCode></NxTable.Cell>
+              <NxTable.Cell>
+                This is added to a list item when the list is in an error state. In this case it's expected that there
+                would only be a single list item which contains the error element.
+              </NxTable.Cell>
+            </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>NxList.Loading</NxCode></NxTable.Cell>
```

Comment:
Also probably better managed via a prop on `NxList` than as a separate component.  Think about how this is going to be used, devs will want to just be able to map a list of data objects to `NxList.Item` components as the children of `NxList`.  Having to manage some other component alongside that list would be annoying.

Reply:
> Also probably better managed via a prop on `NxList` than as a separate component. Think about how this is going to be used, devs will want to just be able to map a list of data objects to `NxList.Item` components as the children of `NxList`. Having to manage some other component alongside that list would be annoying.

`NxList.Loading` has been removed. The loading indicator is now consolidated as a prop on `NxList`.
9c20728ac11326ee3289071ea1c0b76be6c3a745